foxy/ci-nightly-performance.yaml

```diff
@@ -912,7 +912,7 @@ show_plots:
       url: /job/Fci__nightly-performance_ubuntu_focal_amd64/%build%/artifact/ws/test_results/buildfarm_perf_tests/overhead_node_test_results_%name%_resident_anonymous_memory.png
   Performance One Process Test Results (Array1k):
   - title: Average Single-Trip Time
-    description: "The figure shown above shows the average latency time in millisecond for different DDS vendors using a 1K array message. All DDS vendors are configured in asynchronous mode except cycloneDDS which is configured in syncronous mode.<br></br><ul><li><b>QoS</b> Best effort</li><li><b>Rate</b> 1000</li><li><b>Runtime</b> 30 seconds</li><li><b>N process</b> 1</li><li><b>Message size</b> 1K</li></ul>"
+    description: "The figure shown above shows the average single-trip time in millisecond for different DDS vendors using a 1K array message. All DDS vendors are configured in asynchronous mode except cycloneDDS which is configured in syncronous mode.<br></br><ul><li><b>QoS</b> Best effort</li><li><b>Rate</b> 1000</li><li><b>Runtime</b> 30 seconds</li><li><b>N process</b> 1</li><li><b>Message size</b> 1K</li></ul>"
```

Comment:
this change and the remaining ones are valid

Reply:
Actually, the template says latency: https://github.com/ros2/buildfarm_perf_tests/blob/fc9d98d1e6c26c24540869745e55bd0dcb12cb2b/templates/performance_test_1p_1k.txt#L2.

I used the template for generating the foxy file, so I'm pretty sure there's no errors here (i.e. if we want to change something, the template should be changed).

Reply:
updated template https://github.com/ros2/buildfarm_perf_tests/pull/71